jumpscale/entry_points/jsctl.py

```diff
@@ -27,18 +36,20 @@ def list_all():
 
 
 @config.command()
-@click.option("--name")
+@click.argument("name")
 def get(name):
-    value = get_config()[name]
+    _, obj, prop = traverse_config(name)
```

Comment:
Please make sure a value with this key is not there first, because a key like "a.b" can be used on top level, right?

Reply:
Should cases like "a.b": {"b.c": 1} be handled too? And should ambiguities between "a": {"b.c": 1} and  "a.b": {"c": 2} be resolved?